manticore/native/cpu/x86.py

```diff
@@ -583,7 +583,7 @@ def _update_cache(self, name, value):
             self._cache.pop(affected, None)
 
     def read(self, name):
-        name = self._alias(name)
+        name = str(self._alias(name))
```

Comment:
For context, this prevents an issue on line 591, where Python doesn't like evaluating `if 'FLAGS' in name:` if name is not a string. I don't remember how I triggered it, but since the name should already be a string in most reasonable cases, I don't think this will break any functionality.